server/export/exportDetail.js

```diff
@@ -0,0 +1,71 @@
+const Excel = require('exceljs');
+const path = require('path');
+const moment = require('moment');
+const _ = require('lodash');
+const { getUser } = require('../service/userService');
+const {
+  findUserTripsForRange,
+  isDayOfMonth,
+  enumarateTripsWorkingDays,
+  enumarateWorkingDays
+} = require('../service/tripService');
+const {
+  findUserLeavesForRange,
+  enumarateLeavesWorkingDays
+} = require('../service/leaveService');
+
+module.exports = async ({ userId, month }) => {
+  console.log('export to detail', userId, month);
+
+  const user = await getUser(userId);
+  const m_month = moment(month, 'YYYY-MM');
+  const dateRange = [
+    m_month.toDate(),
+    m_month
+      .clone()
+      .add(1, 'month')
+      .toDate()
+  ];
+
+  const isThisMonth = d => isDayOfMonth(d, month);
+  const trips = await findUserTripsForRange(userId, dateRange);
+  const tripDays = enumarateTripsWorkingDays(trips).filter(isThisMonth);
+
+  const leaves = await findUserLeavesForRange(userId, dateRange);
+  const leavesDays = enumarateLeavesWorkingDays(leaves).filter(isThisMonth);
+
+  const monthDays = enumarateWorkingDays(dateRange[0], dateRange[1]);
+  const expectedTickets = _.difference(
```

Comment:
Is a term "ticket" uses as "stravný lístok"? If so, the correct term is "meal voucher".